chapters/synchronization.adoc

```diff
@@ -440,10 +440,11 @@ ifdef::VK_KHR_ray_tracing_maintenance1[]
 endif::VK_KHR_ray_tracing_maintenance1[]
 endif::VK_KHR_acceleration_structure,VK_NV_ray_tracing[]
   * ename:VK_PIPELINE_STAGE_2_ALL_GRAPHICS_BIT specifies the execution of
-    all graphics pipeline stages, and is equivalent to the logical OR of:
+    all graphics pipeline stages, that would be valid in the context that
```

Comment:
"graphics pipeline stages" is somewhat misleading, since it include Bits frome e.g. subpass shading pipeline.